src/styles.less

```diff
@@ -1,33 +1,82 @@
 /* You can add global styles to this file, and also import other style files */
-html, body {
-    direction: ltr;
-    margin: 0;
-    padding: 0;
-    font-size: 18px;
-    height: 100%;
+html,
+body {
+  direction: ltr;
+  margin: 0;
+  padding: 0;
+  font-size: 18px;
+  height: 100%;
 }
 
-body, html {
-    overflow: hidden;
+body,
+html {
+  overflow: hidden;
 }
 
 * {
-    box-sizing: border-box;
+  box-sizing: border-box;
 }
 
 *:focus {
-    outline: none;
+  outline: none;
 }
 
-@import (css) url('https://fonts.googleapis.com/css2?family=Source+Code+Pro:wght@200;400;700&display=swap');
-@import (css) url('https://fonts.googleapis.com/css2?family=Inter:wght@500;700&display=swap');
-@import (css) url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;600;700&display=swap');
-@import (css) '../node_modules/mapbox-gl/dist/mapbox-gl.css';
+@import (css)
+  url("https://fonts.googleapis.com/css2?family=Source+Code+Pro:wght@200;400;700&display=swap");
+@import (css)
+  url("https://fonts.googleapis.com/css2?family=Inter:wght@500;700&display=swap");
+@import (css)
+  url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;600;700&display=swap");
+@import (css) "../node_modules/mapbox-gl/dist/mapbox-gl.css";
 
 @font-face {
-    font-family: 'Manrope';
-    font-style: normal;
-    font-weight: 300 800;
-    font-display: swap;
-    src: url(assets/fonts/manrope.ttf) format('opentype');
+  font-family: "Manrope";
+  font-style: normal;
+  font-weight: 300 800;
+  font-display: swap;
+  src: url(assets/fonts/manrope.ttf) format("opentype");
+}
+.text-content,
```

Comment:
do not add component specific styles here! that's what the component styles are for.